kore/src/Kore/Builtin/AssociativeCommutative.hs

```diff
@@ -260,6 +272,90 @@ instance TermWrapper NormalizedMap where
 
     simplifiedAttributeValue = TermLike.simplifiedAttribute . getMapValue
 
+
+toNormalizedInternalMap
```

Comment:
Can you please document this and the rest of the functions you added to this file?

Reply:
I just did, let me know if you're happy with it.